### PR TITLE
Blockquotes are no longer cut from Gmail emails

### LINKED ISSUE #9: Wrong part of message removed
msg_4vok0e
msg_52774k

### PR DESCRIPTION
**Pull request by divmgl**
Partially fixes https://github.com/quentez/talonjs/issues/9 and https://github.com/frontapp/front/issues/3795

In particular, Gmail `blockquote` tags are no longer removed.

### CODE REVIEW

File: tests/fixtures/nylas/email_15_stripped.html [hunk 1]
```diff
@@ -1 +1,14 @@
...
+<body>
+Test 123<br/><br/><br/>
+
+
+
+
+
+
+</body>
+</html>
```
Comment (divmgl) [inline tests/fixtures/nylas/email_15_stripped.html:14]: Updated from Nylas.

**Comment by quentez** [comment]
@evonck Do you know if that's something we may want to reconsider?

**Comment by evonck** [comment]
@quentez yeah I looked at this one. With the last PR I sent you we don't need this one. This one was only fixing gmail, using the regex first should fix a lot more cases. We might need to reconsider if the perf of the regex are too slow.

**Comment by quentez** [comment]
Ok. Closing then.

**Pull request closed without merging**